site/en/docs/webstore/register/index.md

```diff
@@ -2,35 +2,41 @@
 layout: "layouts/doc-post.njk"
 title: "Register as a Chrome Web Store developer"
 date: 2020-08-19
+updated: 2020-06-27
 description: How to register as a Chrome Web Store developer.
 ---
 
 Before you can publish items on the Chrome Web Store, you must register as a CWS developer and pay a
 one-time registration fee. You must provide a developer email when you create your developer
 account; here are some tips about which email to use:
 
-* It's a good idea to decouple your personal account and your Chrome Web Store publishing account.
-  That's why we suggest using a new email account just for publishing your Chrome Web Store items.
-* If you recently deleted your Chrome Web Store developer account, you can't reuse its associated
+* Because you will receive important emails about your extension or you may want to delete one of your accounts, we suggest using a new email account just for publishing your Chrome Web Store items. 
```

Comment:
Is this the reason that Chary gave for using a separate account?

Reply:
I summarized her response. I'm merging this, but later we can decide if we need to add more details or if this short sentence will suffice.